scripts/Muon/GUI/ElementalAnalysis/PeriodicTable/periodic_table_view.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+
+from PyQt4 import QtCore, QtGui
+
+from silx.gui.widgets import PeriodicTable
+
+
+class PeriodicTableView(QtGui.QWidget):
+    sig_table_changed = QtCore.pyqtSignal(list)
+    sig_table_clicked = QtCore.pyqtSignal(
+        PeriodicTable.ColoredPeriodicTableItem)
+
+    def __init__(self, parent=None):
+        super(PeriodicTableView, self).__init__(parent)
+
+        self.ptable = PeriodicTable.PeriodicTable(self, selectable=True)
+        self.ptable.sigElementClicked.connect(self.table_clicked)
+        self.ptable.sigSelectionChanged.connect(self.table_changed)
+
+        self.grid = QtGui.QGridLayout()
+        self.grid.addWidget(self.ptable)
+        self.setLayout(self.grid)
+
+    def table_clicked(self, item):
+        self.sig_table_clicked.emit(item)
+
+    def table_changed(self, items):
+        self.sig_table_changed.emit(items)
```

Comment:
you might want to add a function similar to https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/FrequencyDomainAnalysis/MaxEnt/maxent_view.py#L155

Reply:
I've seen this in all the widgets, but haven't seen it used anywhere - what's the supposed intended purpose?

Reply:
you are right. I remember correctly its an old method for adding the view to a widget